app/src/constants/privateRouter.tsx

```diff
@@ -2,16 +2,32 @@ import {Navigate} from 'react-router-dom';
 
 import {DISABLE_SIGNIN_REDIRECT} from '../buildconfig';
 import * as ROUTES from './routes';
+import {useGetDefaultToken} from '../utils/tokenHooks';
+import LoadingApp from '../gui/components/loadingApp';
 
 interface PrivateRouteProps {
   // tslint:disable-next-line:no-any
   children: React.ReactElement;
-  allowed: boolean;
 }
 
+/**
+ * Conditional auth route
+ * @param props children to render
+ * @returns Conditionally renders children if token is present (not necessarily
+ * valid) for the first listing entry in the listings db
```

Comment:
@stevecassidy this is an example of where the app has no concept of which token to protect routes against - the idea we discussed of an 'active' profile will solve this problem as we have a definitive token to check for.